vmm/src/seccomp_filters.rs

```diff
@@ -772,12 +774,50 @@ fn api_thread_rules() -> Result<Vec<(i64, Vec<SeccompRule>)>, BackendError> {
     ])
 }
 
+// The filter containing the white listed syscall rules required by the D-Bus API
+// to function.
+#[cfg(feature = "dbus_api")]
+fn dbus_api_thread_rules() -> Result<Vec<(i64, Vec<SeccompRule>)>, BackendError> {
```

Comment:
I think this commit should be split - the renaming one and then separate ones for adding dbus seccomp, etc.

Reply:
I split that commit into three commits, PTAL.